src/standard_library/Prelude.kt

```diff
@@ -0,0 +1,30 @@
+package standard_library
+
+import core.*
+
+// eventually this will be written in tommy script directly
+// but we need to use TAny and Brendan hasn't made the type system good
+
+private fun tprint(message : Expr) = FunCall(Var("print"), listOf(message))
+private fun tpush(arr : Expr, elem : Expr) = FunCall(Var("push"), listOf(arr, elem))
+
+private val defPrintln = FunDef(Var("println"), listOf(AnnotatedVar("message", TAny)), TUnit, listOf(
+        tprint(Var("message")),
+        tprint(LString("\n"))
+))
+
+private val defCons = FunDef(Var("cons"), listOf(AnnotatedVar("list", TArray), AnnotatedVar("elem", TAny)), TArray, listOf(
+        VarDef(AnnotatedVar("newArr", TArray), LArray(mutableListOf())),
```

Comment:
is there going to be a cons operator?

Reply:
idk, I think we used `arr ++ [a]` in one of the test scripts and I figured I'd add it to the stdlib for now

Reply:
I like arr ++ [a]

Reply:
maybe when we are thinking about performance, having ++ [a] temporarily allocates an array which might be annoying. We can prob optimize that away pretty easily though.